examples/atari/atari_dqn.py

```diff
@@ -232,6 +232,7 @@ def watch() -> None:
         sys.exit(0)
 
     # test train_collector and start filling replay buffer
+    train_collector.reset()
     train_collector.collect(n_step=args.batch_size * args.training_num)
```

Comment:
Just wondering: why is this necessary at all? It's not like we are filling the buffer to any significant extent. Does training fail without this? Is it something that should be part of the Trainer implementation?

Reply:
That's a whole different question. I don't think the original DQN paper collected warum up steps before learning. But the line has been in there and unchanged since 2021.

Reply:
That's not even a warm-up. Seems like an arbitrary decision, could you create an issue for that?